Makefile

```diff
@@ -371,10 +366,9 @@ KUSTOMIZE_VERSION ?= v5.0.3
 CONTROLLER_TOOLS_VERSION ?= v0.12.0
 GOLANGCI_LINT_VERSION ?= v1.54.0
 KIND_VERSION ?= v0.20.0
-KUTTL_VERSION ?= 0.15.0
 
 .PHONY: install-tools
-install-tools: kustomize golangci-lint kind controller-gen envtest crdoc kuttl kind operator-sdk
```

Comment:
Can we not install chainsaw locally the same way we did for kuttl?

Reply:
Sure we can do that. What we did in grafana-operator was to support both, only installing it if not already installed.

Reply:
Sure, that works as well.

Reply:
Ok, i'll do that.